tests/contrib/operators/test_segment_track_event_operator.py

```diff
@@ -60,5 +62,28 @@ def test_on_error(self):
             self.test_hook.on_error('error', ['items'])
 
 
+class SegmentTrackEventOperatorTest(unittest.TestCase):
+
+    @mock.patch('airflow.contrib.operators.segment_track_event_operator.SegmentHook')
+    def test_execute(self, mock_hook):
+        user_id = 'user_id'
+        event = 'event'
+        properties = {}
+
+        operator = SegmentTrackEventOperator(
+            task_id='segment-track',
+            user_id=user_id,
+            event=event,
+            properties=properties,
+        )
+
+        operator.execute(None)
```

Comment:
I personally split a test into 3 blocks.

First block for preparation of the objects you want to test.
Second block for the actual operation you want to test.
Third block for assertions.

So I would move the `operator.execute(None)` to the second block.

Just my preference. I am fine with yours, too. :)

Reply:
Updated